graphql/src/execution/execution.rs

```diff
@@ -5,13 +5,14 @@ use graphql_parser::query as q;
 use graphql_parser::schema as s;
 use indexmap::IndexMap;
 use lazy_static::lazy_static;
+use parking_lot::RwLock;
```

Comment:
I am a little concerned about this change, since `parking_lot` changes the policy of how readers and writers interact with each other when the lock is contended. I am not sure though if this will affect us in practice, and philosophically `parking_lot`'s task-fair locking is better than Linux' implementation that might starve writers.

My main concern is how we would spot trouble caused by this in production - it might manifest in a higher p95 for queries. I think we should try it, but be sure to change nothing else so we have a reasonable before/after comparison. I would actually advocate for rolling this one change out in isolation rather than as part of this PR.

Reply:
I've switched back to std RwLock so we can make this change separately.